### DISCUSSION TITLE
Calibre 6.0 add books fails with DeDRM installed

### DISCUSSION THREAD

**Discussion by net-buoy** [apprenticeharper/DeDRM_tools#2093] [General]
per discussion here: https://www.mobileread.com/forums/showpost.php?p=4238241&postcount=106 (error messages included at cited URL) current DeDRM is causing calibre 6.0 to bork on adding books, confirmed by Diap here: https://www.mobileread.com/forums/showpost.php?p=4238157&postcount=99

**Comment by ElleKayEm** [comment]
Fix is in the master branch at noDRM's fork.  Will be in the next release.  (Apprentice Harper has retired.)

**Reply by honglet** [reply]
how may I check out the working revision? Thanks.

**Reply by ElleKayEm** [reply]
Fix is now in the 10.0.3 release:  https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3
